static/js/constants.ts

```diff
@@ -155,6 +155,8 @@ export const exampleSiteConfigFields: ConfigField[] = uniq(
 
 export const CONTENT_TYPE_RESOURCE = "resource"
 export const CONTENT_TYPE_PAGE = "page"
+export const CONTENT_TYPE_COURSE_COLLECTION = "course_collections"
+export const CONTENT_TYPE_RESOURCE_COLLECTION = "resource_collections"
```

Comment:
should we declare these as an enum or a union type here in `constants.ts`? might be nice to have something like a `LinkableResource` type which we can use elsewhere